### PR TITLE
Sprint3

### PR DESCRIPTION
**Pull request by baybiebxm**
Pls, review sprint 3 ka. If you have any questions/bug comment below

### CODE REVIEW

**Comment by sealbb** [comment]
image

this at /backoffice/registrations/4
i think cancle button don't notice me that btn for go to previous page kub just my opinion and this page have only check status action  i think that if we save new status and auto go back to previous page this will be good na

**Comment by sealbb** [comment]
image
when i hover on Going it's present the button can click. if that cursor from my component please create option that not use cursor pointer but if not please use simple cursor tee kub.

** but if that can click in next sprint pls tell me idk kub

**Comment by sealbb** [comment]
image
at /backoffice/registrations/event
for me if i can click anywhere of each row much be better ux and have css when hover

**option up to u kub this one

**Comment by sealbb** [comment]
image

at /backoffice/registrations/event
when hover event name or location it's show a type cursor pls use this 
image

it's give more good feeling

File: components/backoffice/AttendeeDetail.vue [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+              }}
+              <Arrow
+                :class="`${isShowStatusBy ? 'rotate-90' : '-rotate-90'} text-sm duration-200`"
+              />
+            </button>
+            <div
+              v-if="isShowStatusBy"
+              class="absolute top-12 z-20 flex w-full flex-col items-start gap-1 rounded-xl bg-[#fdfdfd] p-3 text-start drop-shadow-lg"
+            >
+              <button
```
Comment (sealbb) [inline components/backoffice/AttendeeDetail.vue:202]: this pls add another cursor design if this can't click kub

**Review by sealbb — CHANGES_REQUESTED** [review]

**Comment by baybiebxm** [comment]
> image
> 
> this at /backoffice/registrations/4 i think cancle button don't notice me that btn for go to previous page kub just my opinion and this page have only check status action i think that if we save new status and auto go back to previous page this will be good na

So, you want me to change the 'Cancel' to another word to tell that 'go to previous page' right? and 'auto go back to previous page' did you try to change the status in the form yet. I have status to tell that 'success or fail'. Think about it or u want me to move that 'status' in a previous page instead
image

**Comment by baybiebxm** [comment]
> image at /backoffice/registrations/event for me if i can click anywhere of each row much be better ux and have css when hover
> 
> **option up to u kub this one

I can change to click the row instead but normally if can click each row, we still have the btn Registration list, is it not strange right

File: components/backoffice/AttendeeDetail.vue (hunk 1 above)
Comment (baybiebxm) [reply components/backoffice/AttendeeDetail.vue:202]: where can't click wa?

Comment (sealbb) [reply components/backoffice/AttendeeDetail.vue:202]: image
Going it can click mai na?

**Comment by sealbb** [comment]
> > image at /backoffice/registrations/event for me if i can click anywhere of each row much be better ux and have css when hover
> > **option up to u kub this one
> 
> I can change to click the row instead but normally if can click each row, we still have the btn Registration list, is it not strange right

mai plak kub

**Comment by sealbb** [comment]
> > image
> > this at /backoffice/registrations/4 i think cancle button don't notice me that btn for go to previous page kub just my opinion and this page have only check status action i think that if we save new status and auto go back to previous page this will be good na
> 
> So, you want me to change the 'Cancel' to another word to tell that 'go to previous page' right? and 'auto go back to previous page' did you try to change the status in the form yet. I have status to tell that 'success or fail'. Think about it or u want me to move that 'status' in a previous page instead image

not sure for word to replace cancle and auto go previous i think if it have popup to tell status ja ok mai but can use same and ignore auto routing kub

**Comment by baybiebxm** [comment]
Changed 'Cancle' to this instead,r u ok with this? and if want to edit some ui talk to me.

image

**Pull request merged**